lib/azure.rb

```diff
@@ -22,6 +22,7 @@
 require 'addressable/uri'
 require 'faraday'
 require 'faraday_middleware'
+require 'azure/core'
```

Comment:
With this change, Should we be removing all the unnecessary explicit requires of individual azure/core files?
Example: [require 'azure/core/signed_service'](https://github.com/Azure/azure-sdk-for-ruby/blob/asm/lib/azure/service_bus/service_bus_service.rb#L15)